shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/algorithm/sharding/inline/InlineShardingAlgorithm.java

```diff
@@ -63,10 +63,15 @@ private boolean isAllowRangeQuery() {
     }
     
     @Override
-    public String doSharding(final Collection<String> availableTargetNames, final PreciseShardingValue<Comparable<?>> shardingValue) {
-        Closure<?> closure = createClosure();
-        closure.setProperty(shardingValue.getColumnName(), shardingValue.getValue());
-        return closure.call().toString();
+    public String doSharding(Collection<String> availableTargetNames, PreciseShardingValue<Comparable<?>> shardingValue) {
```

Comment:
When will that happen NullPointerException? 

NullPointerException var5  maybe  to  (final NullPointerException e)